modules/openapi-generator-cli/src/main/java/org/openapitools/codegen/cmd/CompletionCommand.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2010 the original author or authors.
```

Comment:
Shall we update the above with the following instead?
```
 * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
 * Copyright 2018 SmartBear Software
```

Reply:
@wing328 we can't. Per comment below the header, I took this from the framework's suggest command and renamed it with a description. It's unfortunate that they have such a generic license text, because it makes it look like something that needs to be updated.

Reply:
Oh, or do you think we should add our line as a second line in the header?

Reply:
@wing328 I've added
```
* Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
```
Below the line you've commented on, as this licensed file is originally from airlift/airline, not SmartBear.

Reply:
Looks good to me